content/en/seachat/seachat-manual/02-create-agent/03-agent-duplicaiton.md

```diff
@@ -45,3 +45,36 @@ This process might take some time based on the complexity of the agent. Once you
 <br/>
 
 For users with integrations or channels connected to the original agent, you will need to update the integration to include the new agent copy since the agent copy is still a new agent.
```

Comment:
add a small section called "What's duplicated?" -- basic info, advanced settings, knowledge base, widget, but not channels